src/modules/stats/components/StatsSection.tsx

```diff
@@ -0,0 +1,381 @@
+import { useCallback } from "react";
+import { axiosInstance, HEADERS, TASKS_URLS, USERS_URLS } from "../../../api";
+import useFetch from "../../../hooks/useFetch";
+import StatsCard from "./StatsCard";
+import DoughnutChart from "../../charts/components/DoughnutChart";
+import styles from "./StatsSection.module.css";
+
+interface countTasksType {
+  inProgress: number;
+  toDo: number;
+  done: number;
+}
+const StatsSection = () => {
+  const countTasks = useCallback(async () => {
+    const response = await axiosInstance.get(TASKS_URLS.COUNT_TASKS, HEADERS);
```

Comment:
We can check the correct type for the response from the backend and provide it to the axios method `get<CountTasksResponse>`